PresentScreenings/AppDelegate.designer.cs

```diff
@@ -94,6 +94,9 @@ partial class AppDelegate
 
 		[Action ("ToggleTicketsBought:")]
 		partial void ToggleTicketsBought (Foundation.NSObject sender);
+
+		[Action ("TryShowFilmInfo:")]
+		partial void TryShowFilmInfo (Foundation.NSObject sender);
```

Comment:
Is this used?
The same action is also assigned to the button...
Please remove if superfluous.

Reply:
Could be removed indeed.
Action is passed to the First Responder.